Framework7/features/step_definitions/accounts_steps/accounts_steps.rb

```diff
@@ -22,34 +22,18 @@
 end
 
 Then(/^The response should be the next JSON:$/) do |json_expected|
-  expect(@account_object).to eq(JSON.parse(json_expected))
+  expect(@account_object.to_hash).to eq(JSON.parse(json_expected))
 end
 
-Then(/^The accounts? should have the nex attributes?:$/) do |attribute_types|
-  attribute_hash=Hash.new
-  attribute_types.hashes.each_with_object({}) do |row|
-    attribute_hash[row['attribute']]= row['type']
+Then(/^I expect (true|false) as result of fields? validations?$/) do |is_validated|
+  if @account_object.kind_of?(Array) then
+    @account_object.length.times do |index|
```

Comment:
I think that you should use each_index
http://docs.ruby-lang.org/en/2.0.0/Array.html#method-i-each_index